### ISSUE TITLE
Wishlist: Some kind of Brickify Mesh for 2D

### ISSUE THREAD

**Issue by enzyme69** [nortikin/sverchok#1941]
I have this procedure idea that I actually mentioned at some point.

Wondering if there is algorithm out there that is doing exactly this:
1. Start with 2D shapes, any shapes, including Text
2. In Blender, we can use **Remesh Modifier** for Extruded 2D shapes
3. A simple tiny extrusion from no.2 and Remove Doubles will give us some kind of shapes with grid that is clean

![screen shot 2017-12-17 at 16 26 49](https://user-images.githubusercontent.com/3952687/34076892-50d55746-e347-11e7-9e60-e9e086a99631.png)

Having this "brickify" mesh allow for noise application and displacement that is equal across.

I wonder if this is something that is useful?

Actually to think about it, I think Blender Remesh is actually exactly doing this. But not flat mesh.

**Comment by enzyme69** [comment]
[brickify_002_2017_12_17_05_34.zip](https://github.com/nortikin/sverchok/files/1565342/brickify_002_2017_12_17_05_34.zip)

**Comment by zeffii** [comment]
yeah, I think we could do a set of 2d remesh algorithms, that (unlike the modifier) can return a much nicer outline (identical to the original) but with a nice even spread of verts internally producing tris.
